subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt

```diff
@@ -1,14 +1,11 @@
 package org.gradle.kotlin.dsl.support
 
 import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
-import org.gradle.kotlin.dsl.embeddedKotlinVersion
-
+import org.gradle.kotlin.dsl.*
```

Comment:
Explicit imports?

Reply:
Hmm... Looks like my IDEA configuration adds the `org.kotlin.dsl.*` import automatically. Probably since this is what we do for build scripts?

Reply:
Fixed.